code/datums/quirks/negative.dm

```diff
@@ -242,30 +242,42 @@
 	hardcore_value = 8
 	/// Location of the bottle of pills on spawn
 	var/where
+	var/where_lob
+	var/safety_warning = TRUE
 
 /datum/quirk/brainproblems/on_spawn()
 	var/mob/living/carbon/human/H = quirk_holder
 	var/pills = new /obj/item/storage/pill_bottle/mannitol/braintumor()
+	var/lob = new /obj/item/lobotomizer()
 	var/list/slots = list(
 		LOCATION_LPOCKET = ITEM_SLOT_LPOCKET,
 		LOCATION_RPOCKET = ITEM_SLOT_RPOCKET,
 		LOCATION_BACKPACK = ITEM_SLOT_BACKPACK,
 		LOCATION_HANDS = ITEM_SLOT_HANDS
 	)
 	where = H.equip_in_one_of_slots(pills, slots, FALSE) || "at your feet"
+	where_lob = H.equip_in_one_of_slots(lob, slots, FALSE) || "at your feet"
```

Comment:
Should also try and get the backpack storage treatment like `where` does.

Reply:
Done